tests/e2e/controller_test.go

```diff
@@ -188,7 +188,10 @@ func TestOdhOperator(t *testing.T) {
 				continue
 			}
 
-			t.Run(k, v)
+			t.Run(k, func(t *testing.T) {
+				t.Parallel()
```

Comment:
Wonder if `t.Parellel` should be set in by the individual test suite, because there may be some tests that cannot run in parallel, i.e. the model controller one should not be executed side by side with the `Kserve` or `ModelMesh` one as it does enable/disable those components as part foe the test suite

Reply:
yep :)